chapters/en/slides/module2/module2_29.Rmd

```diff
@@ -58,24 +65,18 @@ Notes: Script here
 
 ---
 
-Let's return to the question we asked at the beginning slides of exercise 25: 
-
-_*Which manufacturer has the highest mean sugar content?*_   
+Let's learn some other useful abilities of `altair`. When specifying `x` and `y` in the `encode(..)` function, altair allows us to specify what data type to expect. See the table below for a description of the data types available and how to specify them.
 
-A nice way of showing our results would be to graph this. A bar chart like this should do the trick! 
+| Data Type   | Shorthand Code | Description |
+| ----------- | ----------- | ----------- |
+| Quantitative | `Q` |  a continuous real-valued quantity |
+| Ordinal   | `O` | a discrete ordered quantity |
+| Nominal | `N` | a discrete unordered quantity |
+| Temporal| `T` | a time or date value |
```

Comment:
I think we agreed on a column for examples? -> https://ubc-mds.slack.com/archives/DGSGPE623/p1596842801114600

I don't love the words "real-valued" is ordinal "fake-valued" then?